mlops_team_project/models/xgboost_model.py

```diff
@@ -42,7 +42,8 @@ class ModelResponse:
 
 BUCKET_NAME = "mlops489-project"
 
-client = storage.Client("mlops489-425700") if os.environ.get('RUN_CML') != "true" else None
+client = storage.Client("mlops489-425700") if (os.environ.get('RUN_CML') != "true"
+                                               and os.environ.get('NO_STORAGE') != "true") else None
```

Comment:
Same thing as the other job, this was throwing an error when the action was run, so I added a specific env var for this

Reply:
For simplicity sake I think you could have just used "NO_STORAGE" for both examples - if we had enough time I'd probably just pass in a cli arg to manage this - but no need to change